app/src/main/java/org/sil/storyproducer/model/Phase.kt

```diff
@@ -1,148 +1,198 @@
+            PhaseType.REMOTE_CHECK     -> R.drawable.ic_school_white_48dp
+            PhaseType.ACCURACY_CHECK   -> R.drawable.ic_school_white_48dp
+            PhaseType.VOICE_STUDIO     -> R.drawable.ic_mic_box_48dp
+            PhaseType.FINALIZE         -> R.drawable.ic_video_call_white_48dp
+            PhaseType.SHARE            -> R.drawable.ic_share_icon_v2_white
             else -> R.drawable.ic_mic_white_48dp
         }
     }
 
+    /**
+     * Get the color associated with a phase
+     * @return color
+     */
+    fun getColor() : Int {
+        return when(phaseType){
+            PhaseType.LEARN -> R.color.learn_phase
+            PhaseType.TRANSLATE_REVISE -> R.color.translate_revise_phase
+            PhaseType.COMMUNITY_WORK   -> R.color.comunity_work_phase
+            PhaseType.WHOLE_STORY      -> R.color.whole_story_phase
+            PhaseType.BACK_T           -> R.color.backT_phase
+            PhaseType.REMOTE_CHECK     -> R.color.remote_check_phase
+            PhaseType.ACCURACY_CHECK   -> R.color.accuracy_check_phase
+            PhaseType.VOICE_STUDIO     -> R.color.voice_studio_phase
+            PhaseType.FINALIZE         -> R.color.finalize_phase
+            PhaseType.SHARE            -> R.color.share_phase
+            else -> R.color.black
+        }
+    }
+
+    /**
+     * Get path of audio file needed by slide, based on the current phase (narration or a draft)
+     * @return String
+     */
     fun getReferenceAudioFile(slideNum: Int = Workspace.activeSlideNum) : String {
         val filename = when (phaseType){
-            PhaseType.DRAFT -> Workspace.activeStory.slides[slideNum].narrationFile
-            PhaseType.COMMUNITY_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.CONSULTANT_CHECK -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.DRAMATIZATION -> Workspace.activeStory.slides[slideNum].chosenDraftFile
-            PhaseType.BACKT -> Workspace.activeStory.slides[slideNum].chosenDraftFile
+            PhaseType.TRANSLATE_REVISE -> Workspace.activeStory.slides[slideNum].narrationFile
+            PhaseType.COMMUNITY_WORK   -> Workspace.activeStory.slides[slideNum].chosenTranslateReviseFile
+            PhaseType.BACK_T           -> Workspace.activeStory.slides[slideNum].chosenTranslateReviseFile
+            PhaseType.ACCURACY_CHECK   -> Workspace.activeStory.slides[slideNum].chosenTranslateReviseFile
+            PhaseType.VOICE_STUDIO     -> Workspace.activeStory.slides[slideNum].chosenTranslateReviseFile
             else -> ""
         }
         return Story.getFilename(filename)
     }
 
-    fun getPrettyName() : String {
+
+     // TODO: refactor
+     // the following 3 functions are used when displaying the phase names and creating paths to store audio files
+     // (see Issue #547)
+
+    /**
+     * Get name for phase to be displayed with help docs

```

Comment:
to be displayed in UI ??   I'm assuming.
I don't know where phase names displayed with help docs. But it's enough to mention "UI" if that's the main thing.

Reply:
Changed this function name to getHelpDocFile. I already changed the HTML files to match the respective phase name changes so we're good here.